kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/WindowDoc.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *
+ *  * Copyright 2023 Responsive Computing, Inc.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package dev.responsive.kafka.internal.db.mongo;
+
+import java.util.Arrays;
+import java.util.Objects;
+import org.bson.codecs.pojo.annotations.BsonId;
+
+public class WindowDoc {
+
+  public static final String ID = "_id";
+  public static final String VALUE = "value";
+  public static final String EPOCH = "epoch";
+  public static final String WINDOW_START_TS = "windowStartTs";
+
+  @BsonId
+  byte[] id;
+  byte[] value;
+  long epoch;
```

Comment:
totally unrelated to this PR, but what do you think about changing epoch to be an int instead of a long? this currently has 8 bytes overhead per row, and MAX_INT gives us 2billion rebalances before running out of room. If we have a rebalance every minute, it would take us 4000 years to hit max int (`2000000000/(60*24*365)`)

Reply:
I had the same thought a while back, I'll file a ticket for it and we can address in a followup PR

Reply:
https://www.notion.so/responsivedev/Consider-changing-epoch-from-long-to-int-b9ab9e5623864b71b4a193162fe842fb?pvs=4